src/openforms/submissions/api/validators.py

```diff
@@ -62,3 +63,24 @@ def __call__(self, form: Form, field: serializers.RelatedField):
                 return
         if form.maintenance_mode:
             raise serializers.ValidationError(self.message, code=self.code)
+
+
+class CheckCheckboxAccepted:
+    def __init__(self, ask_declaration_field_name: str, declaration_display_name: str):
+        self.ask_declaration_field_name = ask_declaration_field_name
+        self.declaration_display_name = declaration_display_name
```

Comment:
I think we can extract this from `model._meta.get_field(...).verbose_name`, but it definitely doesn't hurt to make it explicit